src/components/query-form/query-form.test.js

```diff
@@ -112,33 +112,6 @@ describe('QueryForm', () => {
     })
   })
 
-  describe('when the query changes locally', () => {
```

Comment:
Is there any way to adjust these to make sense with the new plugin? If not, that's okay, just double checking.

Reply:
I think it would be better as a smoke test